cmd/fleet/router.go

```diff
@@ -79,7 +79,7 @@ func NewRouter(bulker bulk.Bulk, ct *CheckinT, et *EnrollerT, at *ArtifactT, ack
 
 	// Install routes
 	for _, rte := range routes {
-		log.Info().
+		log.Debug().
```

Comment:
why make these debug?  useful in production to know specifically what was installed.  we have the unfortunate problem that is near to impossible to turn the server to debug at boot time in cloud so we will never see these.

Reply:
As the routes are hardcoded I expect that always all of them are always running and there is no dynamic part to it. Happy to move it back to Info.

Reply:
Sure.  I know what the routes are cause I wrote them.   I think more about support folks in this case.